nucypher/crypto/ferveo/dkg.py

```diff
@@ -19,14 +19,11 @@
 LOGGER = Logger('ferveo-dkg')
 
 
-class FerveoVariant(Enum):
-    SIMPLE = 0
-    PRECOMPUTED = 1
-
-
 _VARIANTS = {
-    FerveoVariant.SIMPLE: AggregatedTranscript.create_decryption_share_simple,
-    FerveoVariant.PRECOMPUTED: AggregatedTranscript.create_decryption_share_precomputed
+    str(FerveoVariant.simple): AggregatedTranscript.create_decryption_share_simple,
```

Comment:
Do we have to key by string? We can't use the enum instead?

Reply:
:heavy_check_mark: